easybuild/easyconfigs/p/petsc4py/petsc4py-3.21.0-foss-2023b.eb

```
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'petsc4py'
+version = '3.21.0'
+
+homepage = 'https://bitbucket.org/petsc/petsc4py'
+description = "petsc4py are Python bindings for PETSc, the Portable, Extensible Toolchain for Scientific Computation."
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['b2000a3f8ef60920e1f82fa4772372d7941bc737bcc421a234a2507097a44d00']
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('SciPy-bundle', '2023.11'),
+    ('PETSc', '3.21.0'),
```

I assume the versions need to match, don't they?

```suggestion
    ('PETSc', version),
```